main.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-
+import cv2
```

Comment:
`cv2` forms part of `pyvino_utils` so we do not need to import it here.

https://github.com/B0N0AI/pyvino_utils/blob/master/pyvino_utils/input_handler/input_feeder.py#L4
```suggestion
```